### ISSUE TITLE
[Question] Recommendation on how to get PDF versions

### ISSUE THREAD

**Issue by MikeTheWatchGuy** [nngogol/PySimpleGUIDocGen#7]
If I wanted to make a PDF file based on your awesome documentation, what would you suggest to convert the markdown to PDF?  I thought you had some ideas about this the other day.

**Comment by nngogol** [comment]
Yeah, there is `rmarkdown`. It's available on GNU\Linux. It can render markdown in different output formats.

**Comment by nngogol** [comment]
We can also use [this](https://dillinger.io/) - is contains `Export As` function.

**Comment by nngogol** [comment]
currently there are this ways:
![mygraph](https://user-images.githubusercontent.com/25032060/60399469-5e755880-9b6d-11e9-9206-7dc77158b3f5.png)

**Comment by nngogol** [comment]
`pandoc` seems to be a BIG GUY.

**Comment by nngogol** [comment]
dillinger seems to generate pdf, but `fontsize` it pretty big. It's totally not perfect

**Comment by nngogol** [comment]
open in chrome and save as pdf...
`vs code` works too
